src/ConnectedMode/InternalsVisibleTo.cs

```diff
@@ -30,3 +30,14 @@
 [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
 
 #endif
+
+
+
+// TODO - CM cleanup. Remove once we've finished moving files to ConnectedMode and fixed up scopes
```

Comment:
Some of the moved types are `internal` but still currently used by types in `Integration`.

I decided to add this `TODO` and comment for now rather than making the necessary types `public` for now.